bin/alohomora

```diff
@@ -87,6 +106,12 @@ class Main(object):
         parser.add_argument("--idp-name",
                             help="Name of your SAML IdP, as registered with AWS",
                             default=None)
+        parser.add_argument("--aws-partition",
+                            help="Partition of AWS you're using, e.g. `aws-us-gov`",
+                            default=None)
+        parser.add_argument("--alohomora-profile",
+                            help="Name of the alohomora profile to use",
+                            default=DEFAULT_ALOHOMORA_PROFILE)
```

Comment:
Should there be --auth-method arg as the readme shows?

Reply:
I can add one in.  I think the README only shows the config file version, not CLI, but you're right: people will assume they can do both.

Reply:
This was added in d1a6fba